xml/deployment_expert_partitioner.xml

```diff
@@ -15,19 +15,26 @@
   </dm:docmanager>
  </info>
  <para>
-  Sophisticated system configurations require specific disk setups. All common
-  partitioning tasks can be done during the installation. To get persistent
-  device naming with block devices, use the block devices below
-  <filename>/dev/disk/by-id</filename> or
-  <filename>/dev/disk/by-uuid</filename>. Logical Volume Management (LVM) is a
-  disk partitioning scheme that is designed to be much more flexible than the
-  physical partitioning used in standard setups. Its snapshot functionality
-  enables easy creation of data backups. Redundant Array of Independent Disks
-  (RAID) offers increased data integrity, performance, and fault tolerance.
-  &productname; also supports multipath I/O <phrase os="sles"> (see
-  <xref linkend="cha-multipath"/> for details)</phrase>. There is also the
-  option to use iSCSI as a networked disk<phrase os="sles"> (read more about
-  iSCSI in <xref linkend="cha-iscsi"/>)</phrase>.
+  Sophisticated system configurations require specific disk setups. You can
+  perform all common partitioning tasks during the installation.
+ </para>
+ <tip>
+  <para>
+   To get persistent device naming with block devices, use the block devices
+   below <filename>/dev/disk/by-id</filename> or
+   <filename>/dev/disk/by-uuid</filename>.
+  </para>
+ </tip>
```

Comment:
I would be cautious to add an admonition into the abstract. You shouldn't warn the user in the abstract (even if it's a tip). I would remove the surrounding `tip` and just use the paras:

```suggestion
  <para>
   To get persistent device naming with block devices, use the block devices
   below <filename>/dev/disk/by-id</filename> or
   <filename>/dev/disk/by-uuid</filename>.
  </para>
```

Apart from the structural issue, an abstract should give an **overview** of the chapter. IMHO it should not give any recommendations. That is the task of the following sections.